db/models.go

```diff
@@ -25,6 +25,7 @@ type App struct {
 	UpdatedAt    time.Time
 	Isolated     bool
 	Metadata     datatypes.JSON
+	// TODO: add BroadcastedNip47Versions *string. For new apps, should be "1.0 0.0"
```

Comment:
@im-adithya what do you think of this?

So we know what version tag we used when we posted the info event on app creation / update.

Reply:
Yup this sounds good to me, +1 for adding this column.

But will this column have highest supported version ("1.0") or all the supported ones like "1.0 0.0"?

Reply:
I chatted with Bumi and it might just be ok to always re-publish the app info events on startup. Then we do not need to change the database.

CC @frnandu @reneaaron

Reply:
Added code to republish info event